### ISSUE TITLE
Feedback: More chapters in p2

### ISSUE THREAD

**Issue by NielsGade** [loiste-interactive/infra-issues#772]
The chapters are too long + maybe move the autosave at the trap to just before the first c4 explodes, so we don't have to hear the same speech every time

**Comment by madattak** [comment]
Listening to the whole speech again was indeed annoying.

**Comment by Yuccaa** [comment]
Created an issue for the tape autosave. #773

**Comment by sebastiannielsen** [comment]
Yeah agree about the chapters, because its not consistent with Part 1.
Waterplant --> Skyscraper could be a own chapter called "Fragile <something>", like "Fragile Structures" or something like that.

Also it makes sense to have chapter switches there, as theres points of no return at Metroride-->Waterplant and Servicetunnel --> Bunker.

Then part2 would become 3 chapters, which are closer to part1's 4 chapters.
(the first chapter is really not a chapter with one single map)
